<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mesh_Computational_Geometry</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../Images/Captures d’écran/Capture d’écran du 2023-09-11 22-00-32.png</normaloff>../../../../Images/Captures d’écran/Capture d’écran du 2023-09-11 22-00-32.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QFrame" name="frame_3">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="GLDisplayWidget" name="widget" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <widget class="QGroupBox" name="groupBox">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>281</width>
            <height>641</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="title">
           <string>Settings</string>
          </property>
          <widget class="QPushButton" name="exit_button">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>600</y>
             <width>86</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>Exit</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="wireframe_checkbox">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>150</y>
             <width>92</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Wireframe</string>
           </property>
          </widget>
          <widget class="QPushButton" name="load_button">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>110</y>
             <width>101</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>Load mesh</string>
           </property>
          </widget>
          <widget class="QPushButton" name="save_button">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>110</y>
             <width>101</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>Save Mesh</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="inf_point_checkbox">
           <property name="geometry">
            <rect>
             <x>160</x>
             <y>150</y>
             <width>121</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Infinite Point</string>
           </property>
          </widget>
          <widget class="QListWidget" name="object_list">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>261</width>
             <height>51</height>
            </rect>
           </property>
          </widget>
          <widget class="QGroupBox" name="itv_and_cf_box">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>180</y>
             <width>261</width>
             <height>81</height>
            </rect>
           </property>
           <property name="title">
            <string/>
           </property>
           <widget class="QRadioButton" name="it_cf_radio_cf">
            <property name="geometry">
             <rect>
              <x>170</x>
              <y>10</y>
              <width>91</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Circulator</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="it_cf_radio_none">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>81</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>None</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QPushButton" name="it_cf_next_button">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>40</y>
              <width>111</width>
              <height>25</height>
             </rect>
            </property>
            <property name="text">
             <string>Next</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="it_cf_radio_itf">
            <property name="geometry">
             <rect>
              <x>80</x>
              <y>10</y>
              <width>81</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Iterator</string>
            </property>
           </widget>
           <widget class="QPushButton" name="it_cf_prev_button">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>40</y>
              <width>111</width>
              <height>25</height>
             </rect>
            </property>
            <property name="text">
             <string>Prev</string>
            </property>
           </widget>
          </widget>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>280</y>
             <width>261</width>
             <height>111</height>
            </rect>
           </property>
           <property name="title">
            <string>GroupBox</string>
           </property>
           <widget class="QSpinBox" name="numberSplitSpinBox">
            <property name="geometry">
             <rect>
              <x>150</x>
              <y>70</y>
              <width>44</width>
              <height>26</height>
             </rect>
            </property>
           </widget>
           <widget class="QPushButton" name="splitButton">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>70</y>
              <width>101</width>
              <height>25</height>
             </rect>
            </property>
            <property name="text">
             <string>Split Triangle</string>
            </property>
           </widget>
           <widget class="QPushButton" name="color_triangle">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>40</y>
              <width>111</width>
              <height>25</height>
             </rect>
            </property>
            <property name="text">
             <string>Color Triangle</string>
            </property>
           </widget>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Mesh"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionLoad_Mesh">
   <property name="text">
    <string>Load Mesh</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLDisplayWidget</class>
   <extends>QWidget</extends>
   <header>gldisplaywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
